cms/templates/library.html

```diff
@@ -81,7 +81,7 @@ <h3 class="sr">${_("Page Actions")}</h3>
                 <section class="wrapper-xblock level-page is-hidden studio-xblock-wrapper" data-locator="${context_library.location}" data-course-key="${context_library.location.library_key}">
                 </section>
                 <div class="ui-loading">
-                    <p><span class="spin"><i class="icon fa fa-refresh"></i></span> <span class="copy">${_("Loading")}</span></p>
+                    <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
```

Comment:
On second thought, this won't even be announced because it's not in a focusable container. This is purely visual so just the aria-hidden is enough.